src/components/new-request/search.vue

```diff
@@ -109,7 +109,7 @@
           readonly
           filled
           hide-details
-          :label="business.identifier"
```

Comment:
I thought this was cool but UI team didn't want it, so changing back to static label >>>